spec/command_line_ui_spec.rb

```diff
@@ -2,9 +2,22 @@
 require "rspec"
 
 class CommandLineUITest
+  DIMENSION_THREE = 3
+  HUMAN_HUMAN_GAME = "HVH"
   describe CommandLineUI do
```

Comment:
Make sure you're using the latest version of `rspec`. The plain `describe` method is deprecated in the new RSpec.